code/modules/shuttle/shuttles/dropship.dm

```diff
@@ -55,6 +55,14 @@
 /obj/docking_port/mobile/marine_dropship/proc/get_door_data()
 	return door_control.get_data()
 
+/// Is called on_departure and on_arrival to prevent transparent solid multitile airlocks
+/obj/docking_port/mobile/marine_dropship/proc/refresh_multitile_airlock_filler_turfs()
+// Yes, there is 1 tick where you can see through solid airlocks, but it's the simplest solution I could've think of
+	for(var/place in shuttle_areas)
+		for(var/obj/structure/machinery/door/doors in place)
+			if(doors.width > 1)
+				doors.handle_multidoor()
```

Comment:
this pattern ends up iterating through every atom in the world, unfortunately (as areas do not actually keep the stuff in their area in a list).  you should probably put this logic in /obj/structure/machinery/door/onShuttleMove()

Reply:
For some reason if I try to put the logic in onShuttleMove() it doesn't work even though filler_turfs get set properly, afterShuttleMove() however works similarly to this, so I'll be using that.